lib/action_cable/channel/base.rb

```diff
@@ -68,6 +68,8 @@ class Base
       include Callbacks
       include PeriodicTimers
       include Streams
+      include Naming
+      extend Broadcasting
```

Comment:
Sounds to me like that's something the broadcasting concern should handle, so we don't have to say `self.class.broadcasting_for`.